modules/virt-about-upgrading-virt.adoc

```diff
@@ -37,3 +37,13 @@ used to manage the virtual machine process.
 
 * DataVolumes and their associated PersistentVolumeClaims are preserved during
 upgrade.
++
+[IMPORTANT]
+====
+Running virtual machines that cannot be live migrated might block an {product-title} cluster upgrade.
```

Comment:
This first sentence seems grammatically ambiguous, mostly because of how it starts with "Running." Can you make it more explicit? Something like "If you have running virtual machines that cannot be live migrated and you attempt to upgrade your {product-title} cluster, the upgrade might fail." (Just a suggestion, I don't know if that's the best way to phrase it.)

Reply:
Reworded for clarity